src/styles/main.scss

```diff
@@ -4,7 +4,9 @@
 @import "stars";
 
 .container {
-  display: flex;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  gap: 1px 48px;
   justify-content: space-between;
   flex-wrap: wrap;
```

Comment:
Remove flex-wrap, it does not work here anymore, and set justify-content to center.